react/src/contexts/enums/index.tsx

```diff
@@ -0,0 +1,2 @@
+export { default as EnumProvider } from "./EnumProvider";
+export { default as useEnums } from "./useEnums";
```

Comment:
If we want these two to be internal to the `./contexts/enums` module (i.e., never import directly through their individual files but through this file instead), we can drop the `default` from all three files and save some typing. This is optional as you already wrote the code and there's no functional difference.

Reply:
Yeah I'd leave it as it is since I'll replace all this later anyways.